.github/workflows/java-common-release-build-pull-request.yml

```diff
@@ -0,0 +1,175 @@
+name: Release Pull Request
+concurrency:
+  group: release-pr-${{ github.event.pull_request.head.ref }}
+  cancel-in-progress: true
+
+on:
```

Comment:
This one will need to be adapted to workflow_call and all event related variables swapped out to inputs to this workflow.